.ci/no-exception-test.sh

```diff
@@ -116,7 +115,6 @@ no-exception-hadoop-apache-groovy-scouter)
   BRANCH=$(git rev-parse --abbrev-ref HEAD)
   checkout_from https://github.com/checkstyle/contribution
   cd .ci-temp/contribution/checkstyle-tester
-  sed -i'' 's/^guava/#guava/' projects-for-circle.properties
```

Comment:
We need this line
To avoid execution on guava in all jobs
https://github.com/checkstyle/contribution/blob/685f33b3175735b91c403320007e687246dd63b5/checkstyle-tester/projects-to-test-on.properties#L15

Reply:
Fixed, not sure what I was thinking!